__tests__/articles.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require("supertest");
+const app = require("../app");
+const data = require("../db/data/test-data/index");
+const connection = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => connection.end());
+
+describe("GET /api/articles/:article_id", () => {
+  test.only("200: responds with an object of a single article according to the article_id query ", async () => {
+    const response = await request(app).get("/api/articles/9").expect(200);
+    const { article } = response.body;
+    expect(article).toMatchObject({
+      article_id: expect.any(Number),
```

Comment:
I would check that the article_id is 9 here, proving you have the correct article